src/Screens/wishlist/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { NavigationContainer } from "@react-navigation/native";
+import React from "react";
+import {View,Text} from 'react-native'
+import styles from "./styles";
+import {Header,WishlistCard} from "components"
+
+const Wishlist = ({navigation,route}) =>{
+    const item = route.params
+    
+    return (
+    <View style={styles.container}>
+        
+           <Header leftIcon={"arrow-left"} rightIcon={"shopping-cart"} size={28}
+                onLeftIconPress ={()=>{ navigation.navigate('details') }}
+                onRightIconPress ={()=>{ navigation.navigate('') }}
+                image= {false} />
```

Comment:
you can use defaultProps to pass a default value to any prop.

Reply:
yes but all the icons have specific role therefore I have not used that